py3status/modules/diskdata.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""
+Display advanced disk usage information
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds.
+        (default 10)
+    disk: disk whose stat to check. Set to 'all' to get global stats.
+        (default None)
+    format: format of the output.
+        (default "{disk}: {used_percent}% ({total})")
+    format_rate: format for the rates value
+        (default "{value:5.1f} {unit:>4s}")
+    format_space: format for the disk space values
+        (default "{value:3.1f}")
+    sector_size: size of the disk's sectors.
+        (default 512)
+    thresholds: thresholds to use for color changes
+        *(default {'free': [(0, 'bad'), (10, 'degraded'), (100, 'good')],
+        'total': [(0, "good"), (1024, 'degraded'), (1024*1024, 'bad')]})*
+
+Format placeholders:
+    {disk} the selected disk
+    {free} free space on disk in GB
+    {used} used space on disk in GB
+    {used_percent} used space on disk in %
+    {read} reading rate
+    {total} total IO rate
+    {write} writing rate
+
+Rate format placeholders:
+    {unit} name of the unit
+    {value} numeric value of the rate
+
+Color thresholds:
+    {free} Change color based on the value of free
+    {used} Change color based on the value of used_percent
+    {read} Change color based on the value of read
+    {total} Change color based on the value of total
+    {write} Change color based on the value of write
+
+@author guiniol
+@license BSD
+"""
+
+from __future__ import division  # python2 compatibility
+from time import time
+import subprocess
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    disk = None
+    format = "{disk}: {used_percent}% ({total})"
+    format_rate= "{value:5.1f} {unit:>4s}"
```

Comment:
I'd prefer `format_rate= "{value} {unit}"` but saying that I haven't run the module yet so not sure how jumpy it is but I think we can do better on the displaying non jumpy.  But may need some more formatter  fun